doc/appendix-de-kassensichv/receipt-case-definitions/receipt-case-definitions.md

```diff
@@ -0,0 +1,22 @@
+## Receipt Case Definitions
+
+This chapter expands on the definitions of Receipt Cases covered in Chapter ["Receipt Case Definitions"](../../general/receipt-case-definitions/receipt-case-definitions.md) of the General Part, with country specific information applicable to the German market.
+
+### Voucher (Service/Product)
+
+The issuance and redemption of vouchers are processes with DSFinV-K requirements. There are two kinds of vouchers to be considered:
+
+- Single purpose vouchers (DE: Einzweckgutscheine)
+- Multi purpose vouchers (DE: Mehrzweckgutscheine)
+
+The difference is that for single purpose vouchers the VAT rate is known already at issuance because it refers to a single purpose (service or product). On the other hand, in case of multi purpose vouchers the VAT rate is not known because it can be redeemed for different purposes (services and/or products).
+
+Handling the issuance and redemption of single purpose vouchers depends on the type of business that the pos operator runs:
+
+1. If she runs a business with profit determination in regards of **§ 4 Abs. 1 EStG** than she must not consider the VAT rate at issuance of the voucher and the charge item case (`ftChargeItemCase`) `0x4445000000000065` should be used at issuance. At redemption a charge item case considering the VAT rate must be used (one of this: `0x4445000000000069`, `0x444500000000006A`, `0x444500000000006B`, `0x444500000000006C`, `0x444500000000006E` or `0x444500000000006F`). 
+
+2. On the other side, if she runs a business with profit determination in regards of **§ 4 Abs. 3 EStG** than she must consider the VAT as shoon as she issues the voucher. In this case one of the following VAT specific charge item cases can be used for issuance: `0x4445000000000061`, `0x4445000000000062`, `0x4445000000000063`, `0x4445000000000064`, `0x4445000000000066` or `0x4445000000000067`. And for redemption the charge item case `0x444500000000006D`should be used.
+
+The issuance of multi purpose vouchers is covered by the charge item case (`ftChargeItemCase`): `0x4445000000000060`. Since there is no specific product or service coverded by the voucher, the issuance is not taxable and no VAT rate must be specified. Same applies for redemption since the VAT rate is included in the charge items for the products or services of the request. For redemption of a multi purpose voucher, one can use the pay item case (`ftPayItemCase`): `0x444500000000000D` with a positive amount. If the cash register can not handle the redemption within a pay item, then it can alternatively use a charge item having the charge item case (`ftChargeItemCase`): `0x4445000000000068`. 
+
+Furthermore for the issuance of multi purpose vouchers, one can alternatively use pay items instead of charge items. The pay item case (`ftPayItemCase`) to be used is `0x444500000000000D`. Negative amounts get converted to an issuance.
```

Comment:
The usage of the ftPayItemCase schould be the standard. - Therefore, the last paragraph should be changed with the one next to last.

Reply:
Thank you, resolved here: https://github.com/christain-rogobete/interface-doc/commit/f83fb81bad8268214df16ce4622c73df42d4094c